com.unity.render-pipelines.universal/Runtime/ForwardRenderer.cs

```diff
@@ -434,20 +439,13 @@ public override void Setup(ScriptableRenderContext context, ref RenderingData re
                 {
                     if (this.actualRenderingMode == RenderingMode.Deferred)
```

Comment:
For me it would feel a lot cleaner to have a comment that this is only needed for the Forward path and change the condition to `if (this.actualRenderingMode != RenderingMode.Deferred)` and skip the else.

Reply:
Updated the code.